derp/xdp/xdp_default.go

```diff
@@ -26,3 +26,11 @@ func (s *STUNServer) Close() error {
 func (s *STUNServer) Describe(descCh chan<- *prometheus.Desc) {}
 
 func (s *STUNServer) Collect(metricCh chan<- prometheus.Metric) {}
+
+func (s *STUNServer) SetDropSTUN(v bool) error {
+	return errors.New("unimplemented on this GOOS")
+}
+
+func (s *STUNServer) GetDropSTUN() bool {
+	return false
+}
```

Comment:
Maybe it would be more honest to say that it _always_ drops STUN 😄

Reply:
true updated in https://github.com/tailscale/tailscale/pull/12527/commits/05369ddd7b49d22ea9506826f3fbca872bb39c76